src/components/Button/index.js

```diff
@@ -1,22 +1,49 @@
 import React, {PropTypes} from 'react'
 
-const Button = ({
-  children, 
-  onClick, 
-  className,
-}) => (
-  <button
-    className={`button-reset bn dib b0 ttu dim br2 ph3 pv2 white bg-orange tc ${className}`}
-    onClick={onClick}
-  >
-    {children}
-  </button>
-)
+const commonClasses = 'link dib f5 fw6 tracked tc br2 ttu ba pointer'
+
+const sizes = ['large', 'small', 'cta']
+const types = ['success', 'warning', 'danger', 'primary', 'default']
+
+const sizedBtnClasses = {
+  large: 'h3 pa3 eh-btn-primary',
+  small: 'h2 lh-solid pa3 eh-btn',
+  cta: 'lh-solid ph4 pv4 eh-cta'
+}
+
+const solidBtnClasses = {
+  success: 'b--transparent bg-green dark-navy hover-bg-dark-green',
+  warning: 'b--transparent bg-yellow dark-navy hover-bg-orange hover-white',
+  danger: 'b--red bg-red white hover-b--dark-red hover-bg-dark-red',
+  primary: 'b--transparent bg-blue white hover-bg-dark-blue',
+  default: 'b--white bg-white dark-navy hover-bg-light-gray'
+}
+
+const outlineBtnClasses = {
+  success: 'b--green green hover-dark-navy hover-bg-green',
+  warning: 'b--yellow yellow hover-dark-navy hover-bg-yellow',
+  danger: 'b--red red hover-dark-navy hover-bg-red',
+  primary: 'b--blue blue white hover-bg-dark-blue',
+  default: 'b--white white hover-dark-navy hover-bg-white'
+}
+
+const Button = ({href, type='default', size='large', outline=false, children}) => {
+    const btnClasses = outline ? outlineBtnClasses[type] : solidBtnClasses[type]
+    const sizeClasses = sizedBtnClasses[size]
+
+    return (
+      <a href={href} className={`${commonClasses} ${btnClasses} ${sizeClasses}`}>
+        {children}
+      </a>
+    )
+}
 
 Button.propTypes = {
-  children: PropTypes.node.isRequired,
-  onClick: PropTypes.func.isRequired,
-  className: PropTypes.string,
+  href: React.PropTypes.string,
```

Comment:
If you drop `React` and just use `PropTypes` your build error will cease. It is occurring because you import `PropTypes` and never use it.